### PR TITLE
Add (in)active orders endpoint

### PR DESCRIPTION
**Pull request by MaxWinterstein**
Add orders endpoint for active and inactive orders.

Nice to see if there are any open orders, or to see how much stuff you bought recently.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Handle:
 - list stores (`/api/item/`)
 - get a store (`/api/item/:id`)
 - set favorite (`/api/item/:id/setFavorite`)
+- get active orders (`order/vX/active`)
```
Comment (ahivert) [inline README.md:18]: should not we add `/api/` like the others endpoint ?
 ```suggestion
- get active orders (`/api/order/vX/active`)
```

**Review by ahivert — COMMENTED** [review]
Thanks, looks good can you add a simple test for each method added?

File: README.md (hunk 1 above)
Comment (MaxWinterstein) [reply README.md]: changed.

**Comment by MaxWinterstein** [comment]
> Thanks, looks good can you add a simple test for each method added?

- Addes some tests, blindly stolen from the others 😉 
- Added example to readme

**Comment by ahivert** [comment]
Thanks! 0.11.0 published on pypi

**Pull request merged**